eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -144,9 +144,9 @@ function (array $groupIds) {
     public function loadGroupByIdentifier($identifier)
     {
         return $this->getCacheValue(
-            $identifier,
+            $this->escapeForCacheKey($identifier),
             'ez-content-type-group-',
-            function (string $identifier) {
+            function () use ($identifier) {
```

Comment:
Is this change necessary?

Reply:
without this change, we will use the variable after using the escapeForCacheKey function, and we need the original value to search in the database